cmd/builder/builder_options_test.go

```diff
@@ -36,6 +36,7 @@ func TestApplyOptions(t *testing.T) {
 		Options.LoggerOption(zap.NewNop()),
 		Options.MetricsFactoryOption(metrics.NullFactory),
 		Options.MemoryStoreOption(memory.NewStore()),
+		Options.ElasticSearchOption(nil),
```

Comment:
this looks like a coverage-only copout. The test should check that the setting is applied.